fatal: path 'utils.py' exists on disk, but not in 'HEAD'
